<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerWindow</class>
 <widget class="QWidget" name="WorkerWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="numberOfChanging">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>44</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeWorkerButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addWorkerButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>361</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Фамилия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Отчество:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Дата рождения:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Образование:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Вакансия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Зарплата:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Предыдущее место:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Опыт работы:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLineEdit" name="surnameEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lastnameEdit"/>
      </item>
      <item>
       <widget class="QDateEdit" name="birthDateEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="isEducatedEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="vacancyEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="salaryEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lastPlaceEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="experienceOfWorkEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
